src/cmake/externalpackages.cmake

```diff
@@ -208,3 +210,55 @@ endif ()
 
 # end Qt setup
 ###########################################################################
+
+###########################################################################
+# CUDA setup
+
+if (USE_CUDA OR USE_OPTIX)
+    if (NOT CUDAHOME AND NOT $ENV{CUDAHOME} STREQUAL "")
```

Comment:
Yes, I believe you are right. ``CUDAHOME`` was used in a prior incarnation of this file that wasn't using CMake's FindCUDA module, and it might be useful for selecting an alternative CUDA installation at configure time.